gen3-integration-tests/services/graph.py

```diff
@@ -200,7 +211,6 @@ def delete_records(
             if e.response.status_code != expected_status_code:
                 logger.error(f"Error while deleting record: {e.response.text}")
                 raise
-        return result
```

Comment:
why remove the return statement? either put it back here, or delete the returned value usage from `delete_record`: https://github.com/uc-cdis/gen3-code-vigil/blob/e4c5347e6f1cefca89becd80d69569776840f71d/gen3-integration-tests/services/graph.py#L194

Reply:
reverted back the return statement